(dp1
S'output'
p2
S"<type 'exceptions.MemoryError'> "
p3
sS'layer'
p4
S'C:\\Users\\unificace\\unificace\\web2py\\applications\\init\\models\\db.py'
p5
sS'code'
p6
S'<code object <module> at 00000000023467B0, file "C:\\Users\\unificace\\unificace\\web2py\\applications\\init\\models\\db.py", line 14>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File ".\\gluon\\restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "C:\\Users\\unificace\\unificace\\web2py\\applications\\init\\models\\db.py", line 841, in <module>\n    Field(\'device_type\',label=T(\'\xc3\x89quipement\'),requires=IS_IN_SET((\'\',\'T\xc3\xa9l\xc3\xa9phone cellulaire\',\'Cl\xc3\xa9 internet\',\'Tablette LTE\',\'Portable avec carte SIM int\xc3\xa9gr\xc3\xa9e\'))),\n  File "C:\\Users\\unificace\\unificace\\web2py\\gluon\\packages\\dal\\pydal\\objects.py", line 1637, in __init__\n    self.widget = widget\nMemoryError\n'
p11
s.